src/CoreWCF.Primitives/src/CoreWCF/Security/SpnegoTokenAuthenticator.cs

```diff
@@ -156,10 +156,10 @@ private ReadOnlyCollection<IAuthorizationPolicy> GetAuthorizationPolicies(IIdent
             }
             else
             {
-                ClaimsIdentity claimsIdentity = new ClaimsIdentity(remoteIdentity);
+                ClaimsIdentity claimsIdentity = new ClaimsIdentity(remoteIdentity); // why is this ClaimsIdentity not used in the line below ?
```

Comment:
I don't know understand why here we create this identity as it is not used later